web/packages/teleterm/src/ui/Document/Document.tsx

```diff
@@ -34,7 +34,6 @@ const Document: React.FC<{
       tabIndex={visible ? 0 : -1}
       flex="1"
       ref={ref}
-      bg="levels.sunken"
```

Comment:
> However, the background in `Document` is unnecessary (it’s already set on `TabHost`) so I removed it.

Where exactly is it set?

Reply:
Ah sorry, we set it on `body`. But I think it's still fine?

Reply:
Yeah, it should be fine. I just wanted to have a confirmation since I didn't see it in `TabHost`. For posterity: https://github.com/gravitational/teleport/blob/965c98d29c5c357dd536b020adccfff189c0a487/web/packages/teleterm/src/ui/ThemeProvider/globals.ts#L31

Maybe leave a comment saying that the background-color of `Document` is controlled through `<body>` and that it cannot be set on `Document` directly because of Chromium issues? We could link to this PR as well.

Reply:
It's not only fine but should be straight up better, I think we should generally avoid setting bg color where it's not necessary.